### ISSUE TITLE
Good Job with the Launch Program -- Can you add Kiiroo Keon and Oynx + Support?

### ISSUE THREAD

**Issue by hmjkinney** [ZengineerVAM/VAMLaunch#3]
You guys did a great job with the program VAMLaunch. But can you add support for the Oynx + and Kiiroo Keon?

Again great job with this program.

**Comment by qdot** [comment]
Yeah, this needs to be pulled up to the newest version of Buttplug, it's running on an 18 month old version at the moment. It should also be able to use Intiface Desktop so this isn't an issue any longer, heh.

At the moment, I'm doing some work on other libraries to make this easier, but I'm hoping I can circle back to this soon and update it.

**Comment by hmjkinney** [comment]
Cool deal, Thanks again for your fast response and your help with this
issue.

**Comment by qdot** [comment]
Done!

More info at 

https://www.reddit.com/r/VAMscenes/comments/lkx1ui/vamsync_formerly_vamlaunch_now_works_with_kiiroo/

Release at

https://github.com/intiface/vamsync/releases

**Comment by hmjkinney** [comment]
Dude, You are the man, very good job.
